app/views/glossary_terms/index.html.erb

```diff
@@ -8,8 +8,14 @@ tabs = User.current ? [
 @container = :text_image
 %>
 
-<%= content_tag(:div, :glossary_term_index_intro.tp, class: "container-text") %>
-
+<%=
+  content_tag(
+    :div,
+    link_to(:glossary_term_index_documentation.tp,
+    "https://github.com/MushroomObserver/mushroom-observer/blob/main/doc/glossary.md"),
+    class: "container-text"
+  )
+%>
```

Comment:
Note that you can use `tag.div(args)` these days, i'm using it a lot on the bootstrap branch.

Reply:
Good point.  re `tag`